speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/RecorderTests.cpp

```diff
@@ -172,26 +172,28 @@ TEST(RecorderTests, WriteOnlyTime)
     std::string folder = GetTestingDirectory();
     Recorder recorder(TEST_DIRECTORY, "test_recorder_time_only", 0, 0);
     recorder.WriteUserInput(nullptr, 0);
-    recorder.WriteUserInput(nullptr, 0.1);
+    recorder.WriteUserInput(nullptr, 2);
     recorder.WriteUserInput(nullptr, 1);
-    recorder.WriteUserInput(nullptr, -1);
+    recorder.WriteUserInput(nullptr, 6.9);
```

Comment:
Note that user input outside the range of (-1, 1) is undefined behaviour and does not have to be handled correctly by the recorder.